traitsui/editors/video_editor.py

```diff
@@ -40,45 +40,61 @@ class VideoEditor(BasicEditorFactory):
     aspect_ratio = AspectRatio()
 
     #: The name of a trait to synchronise with the audio muted state of
-    #: the video.  The referenced trait should be a Bool.
-    muted = Str(sync_value='from', sync_name='muted')
+    #: the video.
+    muted_name = Str()
+    #: True if the audio is muted, False otherwise
+    muted = Bool(sync_value='from', sync_name='muted_name')
 
     #: The name of a trait to synchronise with the audio volume of
-    #: the video player.  The referenced trait should be a Range(0.0, 100.0).
-    #: Values are on a logarithmic scale.
-    volume = Str(sync_value='from', sync_name='volume')
+    #: the video player.
+    volume_name = Str()
+    #: Audio volume on a logarithmic scale
+    volume = Range(0.0, 100.0, 50.0,
+                   sync_value='from', sync_name='volume_name')
 
     #: The name of a trait to synchronise with the playback rate of
-    #: the video.  The referenced trait should be a Float.
-    playback_rate = Str(sync_value='from', sync_name='playback_rate')
+    #: the video.
+    playback_rate_name = Str()
+    #: The playback speed of the video. Negative values are allowed but may not
+    #: be supported by the underlying implementation.
+    playback_rate = Float(sync_value='from', sync_name='playback_rate_name')
 
     #: The name of a trait to synchronise with the player's state.
-    #: The referenced trait should be a PlayerState.
-    state = Str(sync_value='both', sync_name='state')
+    state_name = Str()
+    #: The state (stopped, playing, paused) of the player
+    state = PlayerState(sync_value='both', sync_name='state_name')
 
     #: The name of a trait to synchronise with the player's position.
-    #: The referenced trait should be a Float representing time in seconds.
-    position = Str(sync_value='both', sync_name='position')
+    position_name = Str()
+    #: The current position, in seconds, in the video.
+    position = Float(sync_value='both', sync_name='position_name')
```

Comment:
For these traits there isn't much use for just providing a default but no sync trait because the editor doesn't provide other ways of doing stop/start/pause or moving through the video, so with the current implementation the user will pretty much _have_ to provide at least something that binds to `state` if anything is going to be played, and some way of setting the position (even if it is only to set it back to the start of the video).

I guess there could be cases where these are controlled via a `Handler` directly manipulating the state of the editor, so I don't know if we need to _require_ these to be bound to names, but I think it's going to be the most common.